src/DynamoCore/Engine/Profiling/NodeProfilingData.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Dynamo.Core;
+using Dynamo.Graph.Nodes;
+using VMDataBridge;
+
+namespace Dynamo.Engine.Profiling
+{
+    class NodeProfilingData : NotificationObject, IDisposable
```

Comment:
can you mark this private? I'd prefer it's explicit for those who are not as familiar with c# accessor rules - but not a big deal if you prefer it this way.

Reply:
After attempting to change to private; "Elements defined in a namespace cannot be explicitly declared as private, protected, protected internal, or private protected". I am assuming that you meant making the class private in this comment.